packages/dotlottie-js/src/tests/lottie-image-node.spec.ts

```diff
@@ -210,13 +213,70 @@ describe('LottieImage', () => {
         expect(uniqueImages.length).toBe(5);
 
         expect(uniqueImages.map((image) => image.fileName)).toEqual([
-          'image_1.png',
-          'image_2.png',
+          'image_1.jpeg',
+          'image_2.jpeg',
           'image_4.png',
           'image_5.png',
           'image_9.png',
         ]);
         expect(uniqueImages.map((image) => image.id)).toEqual(['image_1', 'image_2', 'image_4', 'image_5', 'image_9']);
       });
   });
+
+  it('Properly detects mimetype of images.', async () => {
+    let dotlottie = new DotLottie();
+
+    dotlottie = await dotlottie.fromArrayBuffer(OPTIMIZED_DOTLOTTIE);
+
+    const animations = dotlottie.getAnimations();
+
+    if (animations) {
+      animations.map(async (animation) => {
+        const anim = await animation[1].toJSON({
+          inlineAssets: true,
+        });
+
+        expect(JSON.stringify(anim).includes('image/webp'));
+        expect(!JSON.stringify(anim).includes('image/png'));
+        expect(!JSON.stringify(anim).includes('image/jpeg'));
+      });
+    }
+
+    let bullWithSvg = new DotLottie();
+
+    bullWithSvg = await bullWithSvg.fromArrayBuffer(SVG_IMAGE_DOTLOTTIE);
+
+    const bullAnimations = bullWithSvg.getAnimations();
+
+    if (bullAnimations) {
+      bullAnimations.map(async (animation) => {
+        const anim = await animation[1].toJSON({
+          inlineAssets: true,
+        });
+
+        expect(JSON.stringify(anim).includes('image/webp'));
+        expect(JSON.stringify(anim).includes('image/svg'));
+        expect(!JSON.stringify(anim).includes('image/png'));
+        expect(!JSON.stringify(anim).includes('image/jpeg'));
+      });
+    }
+
+    try {
+      let videoDotLottie = new DotLottie();
+
+      videoDotLottie = await videoDotLottie.fromArrayBuffer(VIDEO_DOTLOTTIE);
+
+      const videoAnimation = videoDotLottie.getAnimations();
+
+      if (videoAnimation) {
+        videoAnimation.map(async (animation) => {
+          await animation[1].toJSON({
+            inlineAssets: true,
+          });
+        });
+      }
+    } catch (error) {
+      expect(error).toBeInstanceOf(Error);
+    }
+  });
```

Comment:
[suggestion]
Consider moving this assertion to an independent test case, as it checks for an error when an unsupported MIME type is used. This differs from the primary intention of the test case, which is to assert the happy path.

Reply:
done @theashraf